pandas/tests/indexes/interval/test_constructors.py

```diff
@@ -291,7 +297,27 @@ def test_left_right_dont_share_data(self):
         assert result._left.base is None or result._left.base is not result._right.base
 
 
-class TestFromTuples(ConstructorTests):
+class TuplesClassConstructorTests(ConstructorTests):
+    @pytest.fixture(
+        params=[
+            ([3, 14, 15, 92, 653], np.int64),
+            (np.arange(10, dtype="int64"), np.int64),
+            (NumericIndex(range(-10, 11), dtype=np.int64), np.int64),
+            (NumericIndex(range(10, 31), dtype=np.uint64), np.int64),
+            (NumericIndex(np.arange(20, 30, 0.5), dtype=np.float64), np.float64),
+            (date_range("20180101", periods=10), "<M8[ns]"),
+            (
+                date_range("20180101", periods=10, tz="US/Eastern"),
+                "datetime64[ns, US/Eastern]",
+            ),
+            (timedelta_range("1 day", periods=10), "<m8[ns]"),
+        ]
+    )
+    def breaks_and_expected_subtype(self, request):
```

Comment:
is this the same as the one above?

Reply:
This can be removed entirely now. I'll update.